### DISCUSSION TITLE
Busco Equipo para Hackaton

### DISCUSSION THREAD

**Discussion by EduGarcia502** [Launch-X-Latam/MisionFrontEnd#584] [Team Posts]
**Que tal buenas noches explorers, busco equipo para backend**
me quede atrazado ya que en estas ultimas semanas he tenido muchos problemas que me an consumido el tiempo para resolverlos, si alguien desea un integrante yo estoy buscando vacante mi correo es lx09922@innovaccion.mx o meduardo.hgarcia@gmai.com

**Comment by rcggomez** [comment]
Hola en mi caso yo no cuento con equipo aun, si gustas podemos generar team? este es mi correo lx04912@innovaccion.mx quedo atento.
Saludos

**Comment by AaronCruzH** [comment]
Hola que tal en mi caso yo tampoco tengo TEAM y ando en busca de uno, mi correo es lx04904@innovaccion.mx

**Comment by floressdi** [comment]
Hola yo tampoco tengo equipo mi correo es 
lx16576@innovaccion.mx

**Comment by jassiel2911** [comment]
Yo igual, estoy con un compa y podemos armar un equipo

Mi correo es lx00601@innovaccion.mx

**Comment by GabomanZX92** [comment]
Hola buen día, yo tampoco tengo equipo, mi correo es lx05217@innovaccion.mx

**Comment by rcggomez** [comment]
@jassiel2911 te envio correo para hacer team? o como nos ponemos en comunicacion.

Saludos

**Comment by jassiel2911** [comment]
Simón amigo, sin problema. :3

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by AlePapaqui** [comment]
Hola @jassiel2911 si todavía tienen lugar en su equipo me podría unir.
Este es mi correo: lx04066@innovaccion.mx

**Comment by Bartt0** [comment]
Hola todavia les falta 1 me podria unir???

**Comment by tintayadev** [comment]
Hola, soy Paulo. Aun necesitan gente para el grupo?? 👀

**Comment by JocelinFlores** [comment]
Hii! aun buscan integrante? tengo conocimiento en HTML, CSS Y Javascript ahh y otro poco de base de datos.

**Comment by Bartt0** [comment]
Si se arma un equipo estoy disponible todavía si gustan formamos 1 entre nosotros

**Comment by shelkanife** [comment]
¡Hola! Estoy disponible para integrarme a algún equipo :)

**Comment by safraZX** [comment]
y tambien busco equipo :c estudio desarrollo estoy en 4 semestre y pues se varias cosas mas alla de web dev

**Comment by neocitylab** [comment]
> y tambien busco equipo :c estudio desarrollo estoy en 4 semestre y pues se varias cosas mas alla de web dev

yo estoy disponible, mi correo es lx10115@innovaccion.mx

**Comment by CarlosFerMont** [comment]
Busco esquipo mi correro es lx16661@innovaccion.mx

**Comment by kerizrh** [comment]
Hola yo igual necesito un equipo  lx07570@innovaccion.mx

**Comment by jassiel2911** [comment]
¿Cuántos deben de ser? jajajaja

Por la escuela no vi las respuestas. :(

Pero aquí ando.
Mañana puedo ya conformar al equipo

**Comment by jassiel2911** [comment]
Si andas disponible todavía responde plox.

Te mando mi número en cuanto me confirmes y armemos el grupo

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by joseandresgavilanes** [comment]
Necesito equipo
